kentikapi/errors/errors.go

```diff
@@ -0,0 +1,231 @@
+package kentik_errors
+
+import (
+	"errors"
+	"fmt"
+	"net/http"
+
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+type Code int
+
+const (
+	AuthError          Code = 0
+	InvalidRequest     Code = 1
+	NotFound           Code = 2
+	RateLimitExhausted Code = 3
+	Temporary          Code = 4
+	Timeout            Code = 5
+)
+
+type KentikError struct {
+	Codes []Code
```

Comment:
```suggestion
	code Code
```
I don't see a reason to hold multiple error codes. For example, RateLimitExhausted is temporary error, so there is no need to mark it also as Temporary.

Reply:
My knowledge about this task is based on https://docs.google.com/document/d/1b-huL-uHSVyI3eaao3-YOFHOZ115Sz9FOjlNOdHIULY/edit#heading=h.6hi58cjyboxo
To be right with the error table from this document I decide to keep error codes in slice.

Reply:
Done